### PR TITLE
fix: respect `formats` option when using external config

### LINKED ISSUE #184: Array values in config file are being merged into the defaults
If you use a config file and try to set the formats option to something other than the default, webfont will still generate the default set of font formats. 

The following line seems the culprit:

https://github.com/itgalaxy/webfont/blob/6bacf7697a6d5acd6df038ce0fb6c20fe9fcb2fc/src/standalone.js#L218

`deepmerge()` concats the formats array from the config file with the default array instead of replacing it, which results in the following: `formats: ['svg', 'ttf', 'eot', 'woff', 'woff2', 'svg', 'ttf', 'eot']`

### PR DESCRIPTION
**Pull request by grelas**
This PR fixes https://github.com/itgalaxy/webfont/issues/184 where the `formats` option was not being overwritten when using a config file.

### CODE REVIEW

File: src/standalone.js [hunk 1]
```diff
@@ -266,9 +268,7 @@ export default async function(initialOptions) {
 
       nunjucks.configure(path.resolve(__dirname, "../"));
 
-      templateFilePath = `${templateDirectory}/template.${
-        options.template
-      }.njk`;
+      templateFilePath = `${templateDirectory}/template.${options.template}.njk`;
```
Comment (grelas) [inline src/standalone.js:271]: This change is from `yarn prettify`

**Comment by grelas** [comment]
@evilebottnawi Any chance you could review?

File: src/standalone.js (hunk 1 above)
Comment (jimmyandrade) [reply src/standalone.js:271]: No prob 👍

**Review by jimmyandrade — APPROVED** [review]
LGTM

**Comment by evilebottnawi** [comment]
@jimmyandrade Feel free to merge, I don't have many time on this project, so feel to go ahead

**Comment by jimmyandrade** [comment]
> @evilebottnawi Any chance you could review?

Hello, I'm reviewing your code right now. Thanks for contributing 😄

**Comment by jimmyandrade** [comment]
> @jimmyandrade Feel free to merge, I don't have many time on this project, so feel free to go ahead

Thanks @evilebottnawi. I just merged this PR.
How could we publish this so that @lepus can test? Are there any recommendations?

**Pull request merged**